README.md

```diff
@@ -160,14 +162,14 @@ While the Swift script above is functionally equivalent to the shell script, it
 
 ## Proposed solution
 
-We propose a new type, `struct Subprocess`, that will eventually replace `Process` as the canonical way to launch a process in `Swift`.
+We propose a new package, `Subprocess`, that will eventually replace `Process` as the canonical way to launch a process in `Swift`.
 
-Here's the above script rewritten using `Subprocess`:
+Here's the above script rewritten using `Subprocess` package:
 
 ```swift
 #!/usr/bin/swift
 
-import FoundationEssentials
+import Subprocess
 
 let gitResult = try await Subprocess.run(   // <- 0
```

Comment:
```suggestion
let gitResult = try await run(   // <- 0
```

Reply:
On in testing I found you can't actually omit the `Subprocess` here otherwise you get
```
Use of 'run' refers to instance method rather than global function 'run(_:arguments:environment:workingDirectory:platformOptions:input:output:error:)' in module 'Subprocess'
```

Reply:
^^ That only existed under `XCTestCase` because of the name `run()`.